### DISCUSSION TITLE
Shitposting until I escape from Cthulhu

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#2] [General]
![](https://external-preview.redd.it/swtZ-vPZqmNq8O--KGvc2QA0oJfH21uf0N3umZC0Q40.png?format=pjpg&auto=webp&s=20907408c0ce5986baf5dc8c827986f47834e15e)

**Comment by likeproblem** [comment]
![](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQvhrlLZ56CyqXpUUmeXqJDzhJrggaaoGdC2dEoLRm3rmr-g2BrkuGtVpOqVLQGS6jZKyY&usqp=CAU)

**Comment by likeproblem** [comment]
![](https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/0d1eabb2-1302-44ba-b027-2af87ee4c033/d8fgo1b-782c6678-42da-4442-9c7b-081e1a9938b8.jpg/v1/fill/w_877,h_911,q_70,strp/kerbal_space_program__related_meme_xd__by_solargaming_d8fgo1b-pre.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9MTA2MyIsInBhdGgiOiJcL2ZcLzBkMWVhYmIyLTEzMDItNDRiYS1iMDI3LTJhZjg3ZWU0YzAzM1wvZDhmZ28xYi03ODJjNjY3OC00MmRhLTQ0NDItOWM3Yi0wODFlMWE5OTM4YjguanBnIiwid2lkdGgiOiI8PTEwMjQifV1dLCJhdWQiOlsidXJuOnNlcnZpY2U6aW1hZ2Uub3BlcmF0aW9ucyJdfQ.N1lcipxWhHT34tPC6npjYD4CPNSrvCn6mVQW1FS8U4E)

**Comment by likeproblem** [comment]
![](https://i.imgur.com/yEHNKZ1.jpg)

**Comment by UncleMateo** [comment]
Every time I try to post an update on the dang forums, it goes Error504 on me!
Jeez..

Anyways I'm working on the big list and also on some parts
hopefully not the same ones you're working on lol
![my face](https://github.com/user-attachments/assets/be190572-63f6-4727-99f9-0a4f11ea8171)

**Reply by likeproblem** [reply]
> Every time I try to post an update on the dang forums, it goes Error504 on me! Jeez..
> 
> Anyways I'm working on the big list and also on some parts hopefully not the same ones you're working on lol ![my face](https://private-user-images.githubusercontent.com/178738632/375082157-be190572-63f6-4727-99f9-0a4f11ea8171.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0OTc5MzUsIm5iZiI6MTcyODQ5NzYzNSwicGF0aCI6Ii8xNzg3Mzg2MzIvMzc1MDgyMTU3LWJlMTkwNTcyLTYzZjYtNDcyNy05OWY5LTBhNGYxMWVhODE3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxODEzNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MDZiYmMyZWFmNmFkODk5ZTY3NDY0NjgwYThlM2ZlNjFmMjlkZTk1ZDBkZTkzMTVlYTZkMTQ0YjNiMzFjZWI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JhXndw0iKQe0oiiNJqit-k0z6cON4aat2PQeUg5eicI)

Oh hi! So Im right now not working on anything, but just in case I will create wiki page with parts we both are working on so we don't do the same ones lol

**Comment by likeproblem** [comment]
Almost forgot about my daily streak
![](https://i.imgur.com/A4Z2MDZ.jpg)

**Comment by likeproblem** [comment]
![](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTHLEjs-mGcdwocfpWT1_b8ulxC7qj-jKh0NApg7RKVyXIZ5uzBcn3cW05PrbkIUJUC6fM&usqp=CAU)

**Comment by likeproblem** [comment]
![](https://images7.memedroid.com/images/UPLOADED64/53af3c372d0b6.jpeg)

**Comment by likeproblem** [comment]
![](https://i.imgflip.com/6xjrqc.jpg)

**Comment by likeproblem** [comment]
No I broke my streak :(
